pithos/plugins/_mpris.py

```diff
@@ -576,6 +581,17 @@ def CanEditTracks(self):
         '''b Read only Interface MediaPlayer2.TrackList'''
         return False
 
+    @dbus_property(MEDIA_PLAYER2_RATINGS_IFACE, signature='b')
+    def CanRate(self):
```

Comment:
Here's what the interface looks like in Gjs.
https://gist.github.com/JasonLG1979/77c6f03cc1bb23fabc78394f4be31332